opencog/cogserver/scm/config.scm

```diff
@@ -44,6 +44,7 @@
 (add-to-ltdl-path (string-append (getcwd) "/opencog/nlp/sureal"))
 (add-to-ltdl-path (string-append (getcwd) "/opencog/nlp/lg-dict"))
 (add-to-ltdl-path (string-append (getcwd) "/opencog/planning"))
+(add-to-ltdl-path (string-append (getcwd) "/opencog/scm/opencog/learning"))
```

Comment:
this cannot possibly be right, since there is no ltdl in there.  In fact, this entire section appears to be incorrect, as there are no ltdl's located at any of these paths.  This entire portion of this file needs to be removed.